bin/subzero/node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "subzero-node"
-version = "3.2.62"
+version = "3.2.63"
```

Comment:
Update runtime cargo version as well.

Reply:
Why we should increment the version for Zero even if there were no changes?
As for standalone, I still don't see any reason of supporting it at all.